package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@dapplion/benchmark",
-  "version": "1.0.0",
-  "repository": "git@github.com:dapplion/benchmark.git",
-  "author": "dapplion <35266934+dapplion@users.noreply.github.com>",
+  "name": "@chainsafe/benchmark",
+  "version": "1.1.0.rc-0",
```

Comment:
The only key factor is the version number. As we forked the repo from the version 1.0.0 originally, But it's a newer package completely, should we use `1.1.0` or start from zero versioning `0.1.0`?

Reply:
it's stable enough I would assume to go with `1.1.0` or even `1.0.0` since it's a new package

Reply:
There are lot of thoughts behind zero versioning scheme. https://0ver.org/